.travis.yml

```diff
@@ -6,13 +6,14 @@ services:
 branches:
   only:
     - qa
+    - more-vega-overrides
```

Comment:
Should this be removed before merging?

Reply:
Yes, will remove it. There's no easier way to test this on a third-party site like Vega without deploying it to our own QA instance :(